Casks/f/find-any-file.rb

```diff
@@ -1,16 +1,16 @@
 cask "find-any-file" do
-  version "2.5.5"
-  sha256 "57ef75b7ee92e9d27a570d4e878e4e1ccd2805456411f089bf0459224c555f2b"
+  version "2.5.5,394"
+  sha256 :no_check
 
-  url "https://s3.amazonaws.com/files.tempel.org/FindAnyFile_#{version}.zip",
+  url "https://s3.amazonaws.com/files.tempel.org/FindAnyFile_#{version.csv.first}.zip",
       verified: "s3.amazonaws.com/"
   name "Find Any File"
   desc "File finder"
   homepage "https://findanyfile.app/"
 
   livecheck do
-    url :homepage
-    regex(/href=.*?FindAnyFile[._-]v?(\d+(?:\.\d+)+)\.zip/i)
+    url "https://findanyfile.app/appcast#{version.csv.first.major}.php"
```

Comment:
```suggestion
    url "https://findanyfile.app/appcast#{version.major}.php"
```
This should still work here